### PR TITLE
Move actionx enums to separate compilation unit

### PR DESCRIPTION
**Pull request by joakim-hove**
IN order to avoid circular references in the actionx restart code

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this please

File: src/opm/parser/eclipse/EclipseState/Schedule/Action/Enums.cpp [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+namespace Opm {
+namespace Action {
+
+#include <opm/parser/eclipse/EclipseState/Schedule/Action/Enums.hpp>
```
Comment (akva2) [inline src/opm/parser/eclipse/EclipseState/Schedule/Action/Enums.cpp:23]: will this get code later? as is this cpp file is pointless.

**Comment by akva2** [comment]
jenkins build this please

File: src/opm/parser/eclipse/EclipseState/Schedule/Action/Enums.cpp (hunk 1 above)
Comment (joakim-hove) [reply src/opm/parser/eclipse/EclipseState/Schedule/Action/Enums.cpp:23]: Will get code yes.

**Pull request closed without merging**